services/device-registry-base/src/main/java/org/eclipse/hono/service/management/tenant/TrustedCertificateAuthority.java

```diff
@@ -339,4 +344,30 @@ public final TrustedCertificateAuthority setAutoProvisioningDeviceIdTemplate(fin
         this.autoProvisioningDeviceIdTemplate = template;
         return this;
     }
+
+    /**
+     * Gets the template for generating the authentication identifier.
+     *
+     * @return the template for generating the authentication identifier.
+     */
+    public String getAuthIdTemplate() {
+        return this.authIdTemplate;
+    }
+
+    /**
+     * Sets the template for generating the authentication identifier.
+     * <p>
+     * The validity of the template is verified using {@link IdentityTemplate#checkValidity(String)}.
+     *
+     * @param template the template for generating the authentication identifier.
+     * @return A reference to this for fluent use.
+     * @throws NullPointerException if the template is {@code null}.
+     * @throws IllegalArgumentException if the template is not valid.
+     */
+    public TrustedCertificateAuthority setAuthIdTemplate(final String template) {
```

Comment:
I assume you expect this method to be invoked by Jackson when unmarshalling a JSON request body, right?
I am not sure if this is actually the case because you have annotated the authIdTemplate field with `@JonsProperty` instead of this setter. You should validate by means of adding a corresponding test case to `TenantTest`

Reply:
As per my understanding, even though a field is annotated, the getter and setter method  (following the correct naming convention) are used for marshalling and unmarshalling of JSON by Jackson. I have also added a corresponding test in `TenantTest`.